new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItElasticLogging.java

```diff
@@ -102,8 +121,14 @@
   private static LoggingExporterParams kibanaParams = null;
   private static LoggingFacade logger = null;
 
+  private static String k8sExecCmdPrefix;
+  private static Map<String, String> testVarMap;
+  private static final int maxIterationsPod = 10;
+
   /**
-   * Install Elasticsearch, Kibana and Operator and verify, create a one cluster domain.
+   * Install Elasticsearch, Kibana and Operator and verify,
```

Comment:
Can u re-phrase the description

Install Elasticsearch, Kibana and Operator.
Install WebLogic logging exporter in all WebLogic server pods to collect WebLogic logs.

[create a one cluster domain.  does not make sense  ]